Automattic/woocommerce-payments
# Fix price on payment request for variable subscription products

Fixes #3536 

For variable subscription products the payment request on Product page was using base product price. Fixed the condition for getting the variation pricing, which did not check for 'variable-subscription'.

#### Changes proposed in this Pull Request

- added check for 'variable-subscription' so that variation price calculation is not skipped for subscription variations

#### Testing instructions

 - Go to a Woo product page that has a subscription product with variation, each with a different pricing.
 - Select a non-default option from the product drop-down 
 - Click G Pay or Apple Pay. 
 - The correct variation price should be shown in the Apple Pay / G Pay preview

-------------------


**Post merge**
